docs/design/concurrent-merge-split.md

```
@@ -0,0 +1,321 @@
+
+### Non-Goals
+
+- Undo/redo support for merge/split (deferred per `undo-redo.md` Phase 2).
+- General-purpose `Tree.Move` operation (Phase 2).
+- JS SDK changes (separate follow-up).
+
+## Tree.Edit Convergence Coverage
+
+### Edit execution flow
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)